app/pods/publications/publication/publishpreview/template.hbs

```diff
@@ -78,10 +78,7 @@
         <div class="auk-o-grid">
           <div class="auk-o-grid-col auk-o-grid-col-bp2-3 auk-u-mb-3">
             <div class="auk-form-group">
-              <WebComponents::AuLabel>{{t "publication-form-numac-nummer"}}</WebComponents::AuLabel>
-              {{!--TODO remove the readonly field --}}
-              {{!--Value moet nog komen vanaf het model (activity) --}}
-              <WebComponents::AuInput readonly></WebComponents::AuInput>
+              <span class="vl-icon--light">{{t "no-numac-number-yet"}}</span>
```

Comment:
Deze zou ik liever herwerkt hebben naar een auk versie. Wanneer de icon dependency branch doorgetrokken wordt, dan kan alle icon-related css eruit. 

Ofwel moeten we dit dan opnemen en aanpassen of we passen het nu aan en besparen ons mogelijks een bug in de toekomst (we gaan dit sowieso vergeten)